package.json

```diff
@@ -2,67 +2,86 @@
   "name": "fiscal-data-package-viewer",
   "version": "0.0.1-alpha",
   "description": "A frontend data/views app powered directly from a Fiscal Data Package.",
-  "main": "src/scripts/index.js",
+  "keywords": [],
+  "main": "index.js",
   "scripts": {
-    "build": "webpack --progress --colors",
-    "develop": "webpack-dev-server --progress --colors --watch",
-    "review": "jscs src"
+    "start": "node index.js",
+    "test": "mocha tests/*.js",
+    "review": "jscs app tests",
+    "develop": "gulp && npm start",
+    "build": "gulp"
+  },
+  "engines": {
+    "node": "^4.1.0",
+    "npm": "^2.14.0"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/openspending/fiscal-data-package-viewer.git"
   },
-  "keywords": [
-    "fiscal",
-    "data",
-    "openspending"
+  "author": {
+    "name": "Open Knowledge",
+    "email": "info@okfn.org",
+    "url": "https://okfn.org/"
+  },
+  "contributors": [
+    {
+      "name": "Levko Kravets",
+      "email": "kravets@qarea.com"
+    }
   ],
-  "author": "Open Knowledge",
   "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/openspending/fiscal-data-package-viewer/issues"
-  },
+  "bugs": "https://github.com/openspending/fiscal-data-package-viewer/issues",
   "homepage": "https://github.com/openspending/fiscal-data-package-viewer#readme",
   "dependencies": {
-    "babel-core": "^6.1.4",
-    "babel-loader": "^6.2.0",
-    "babel-polyfill": "^6.1.4",
-    "babel-preset-es2015": "^6.1.4",
-    "babel-preset-react": "^6.1.18",
-    "babel-runtime": "^6.3.13",
+    "angular-filter": "^0.5.8",
+    "angular-ui-bootstrap": "^1.1.2",
+    "babbage.ui": "0.0.1-alpha",
     "bluebird": "^3.0.5",
-    "bootstrap-sass": "^3.3.5",
-    "d3": "^3.5.9",
+    "body-parser": "^1.14.1",
+    "bubbletree": "git+https://github.com/okfn/bubbletree.git",
+    "c3": "git+https://github.com/masayuki0812/c3.git",
+    "d3": "^3.5.14",
+    "datapackage-registry": "^0.1.0",
+    "datapackage-validate": "git+https://github.com/okfn/datapackage-validate.git",
     "express": "^4.13.3",
-    "fiscaldata-js": "git+https://github.com/openspending/fiscal-data-js",
-    "font-awesome": "^4.4.0",
-    "isomorphic-fetch": "^2.2.0",
-    "lodash": "^3.10.1",
-    "node-sass": "^3.4.2",
+    "font-awesome": "^4.5.0",
+    "goodtables": "git+https://github.com/okfn/goodtables-js.git",
```

Comment:
@borysyuk looks like you have just copied in everything from the Packager. Please do a proper clean up and remove all deps and code that does not belong to viewer.